.github/bin/spellcheck.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# The markdown mode of aspell does not make code be ignored. This sed command
+# removes both inline code and code blocks.
+sed -e 's/\s*`\b.*\b`//g' -e '/```.*/,/```/d' README.md -i
```

Comment:
Nice;

    do we want to handle indented markdown as well?

Reply:
Hmm, we can't just delete indented lines because it may just be the second line of a list or something... I think I would take issue in a review if someone mixes code block styles anyway, so I don't think it would be a problem.

Reply:
Ok, I'm not 100% sure I'd agree that the two different block syntaxeis are conflicting with each other, but I'm fine with it for the purposes of this change.

Reply:
If you have a good regex, we could add it. 

"Delete line if it is the first one in a block and it is indented, repeat until the whole paragraph is gone". This depends on the next line being seen as the first after the first is deleted. Schematic sed script:

```sed
/^$/n
/^\t/d
```

Reply:
maybe `sed '/\n^\t/,/^$/d'`

Reply:
I wouldn't rely on \t; I suppose markdown maybe can support it? But formally, I thought the line prefix would be four literal spaces, '`    `'.

sed is line based, so will not be able to match `/\n^/`. How about just `s/^    .*//`?

Reply:
I reread what you said, and i completely missed your point (I don't know how).

We would like the following to be ignored by a spellcheck:

```markdown
    if (true)
        foo(bar, baz)
```

while still enforcing spelling on

```markdown
* foo
  * bar
    * baz 
```

(The last line matches `/^    /`, but we want to make sure baz is spelled correctly.)

I think you should skip solving this, at least as part of this PR. Future improvement!

Reply:
Yeah, that last case is what I had in mind. I'll resolve then.